tsconfig.json

```diff
@@ -13,6 +14,9 @@
       "angular-instantsearch": ["../src/index"]
     }
   },
+  "angularCompilerOptions": {
+    "disableTypeScriptVersionCheck": true
```

Comment:
are you sure we want to do this?

Reply:
I did this for now, since I wasn't sure how much the angular compiler could be updated without breaking backwards compatibility